debian/changelog

```diff
@@ -1,3 +1,13 @@
+flatpak (1.6.2-1endless1) master; urgency=medium
```

Comment:
Historically we've had `eos` here where you have `master`. Is that intentional?

Reply:
Using `eos` is wrong, IMO. Our tools don't actually care, but what's listed there is the codename of distro that's being targeted. E.g., debian doesn't put `debian` there. They use `unstable` or `buster` (for example) if they were targeting a stable branch. Equivalently, if you were targeting an EOS stable branch instead of master, you'd want to put `eos3.7` there or something. I think I documented that in the wiki, but probably not.

Reply:
Okay